lib/mixins/flowdock.ts

```diff
@@ -0,0 +1,21 @@
+import * as request from 'request-promise';
+import { FlowdockInboxItem } from './flowdock-types';
+
+export class FlowdockAdapter {
+    public postToInbox(item: FlowdockInboxItem) {
+        //  Make a straight post to the inbox from the client.
+        const requestOpts = {
+            body: item,
+            json: true,
+            url: `https://api.flowdock.com/messages/team_inbox/${item.roomId}`,
+        };
+
+        // If we aren't outputting to Flowdock (development build, test build, etc)
+        // just return.
+        // People invariably use different cases, so...
+        if (process.env.FLOWDOCK_ALERTS && (process.env.FLOWDOCK_ALERTS.toLowerCase() === 'true')) {
+            // We don't wait for a response, we have no control over it anyway.
+            request.post(requestOpts);
```

Comment:
For this post request though, we don't do anything I think? It'll never reach `reportError`, because we don't return the promise, and we don't `catch` it here either, so any failure to talk to flowdock disappears into the ether. Or am I missing some other connection between this and `reportError`?